test/testnet/single/AccountingEngine.t.sol

```diff
@@ -217,7 +217,7 @@ contract SingleAccountingEngineTest is DSTest {
     _popDebtFromQueue(200 ether);
     safeEngine.createUnbackedDebt(address(0), address(accountingEngine), rad(100 ether));
 
-    assertTrue(can_auction_debt());
+    assertTrue(!can_auction_debt());
```

Comment:
Can you confirm this should be false here?

Reply:
can confirm.  it should be false.  it's testing [this revert.](https://github.com/open-dollar/od-contracts/blob/c7371d525422c18b0ce96d4778fc2aa5c9fc9feb/src/contracts/AccountingEngine.sol#L183)